hooks.lua

```diff
@@ -13,6 +13,8 @@ function InitHooks(a_Plugin)
 	cPluginManager:AddHook(cPluginManager.HOOK_PLAYER_SPAWNED,     OnPlayerSpawned)
 	cPluginManager:AddHook(cPluginManager.HOOK_TAKE_DAMAGE,        OnTakeDamage)
 	cPluginManager:AddHook(cPluginManager.HOOK_WORLD_TICK,         OnWorldTick)
+    cPluginManager:AddHook(cPluginManager.HOOK_PLAYER_TOSSING_ITEM,OnPlayerTossingItem)
```

Comment:
Could you make this a tab and make sure `OnPlayerTossingItem` aligns with the other hooks? If the hook gets so long that you can't add a space after the comma you can add one space to the other hooks.